src/box/tuple_format.c

```diff
@@ -166,12 +166,25 @@ tuple_field_delete(struct tuple_field *field)
 
 /** Return path to a tuple field. Used for error reporting. */
 static const char *
-tuple_field_path(const struct tuple_field *field)
+tuple_field_path(const struct tuple_field *field, const struct tuple_format *format)
 {
 	assert(field->token.parent != NULL);
 	if (field->token.parent->parent == NULL) {
-		/* Top-level field, no need to format the path. */
-		return int2str(field->token.num + TUPLE_INDEX_BASE);
+		int n_named = format->dict->name_count;
+
+
```

Comment:
No need to use so many empty lines. We don't use more than 1 empty line in a row.